tensorflow/lite/c/common.h

```diff
@@ -738,6 +738,8 @@ typedef struct TfLiteContext {
   // WARNING: This is an experimental interface that is subject to change.
   void* (*AllocatePersistentBuffer)(struct TfLiteContext* ctx, size_t bytes);
 
+  void* (*AllocateTempBuffer)(struct TfLiteContext* context, size_t bytes);
```

Comment:
Removed AllocateTempBuffer  API as it is not used in our code.
Used GetScratchSafe API and that calls  GetScratchTensor as part of prepare of unidirectional LSTM  to get  scratch buffer, and we are not sure if this can be replaced with RequestScratchBufferInArena